src/sensors/heading/gyroscope/GY50.cpp

```diff
@@ -113,9 +111,7 @@ int GY50::getAngularVelocity()
     auto firstByte  = readL3G4200DRegister(zAxisFirstByteRegister);
     auto secondByte = readL3G4200DRegister(zAxisSecondByteRegister);
 
-    static constexpr auto kBitsInByte = 8;
-
-    return static_cast<int16_t>((firstByte << kBitsInByte) | secondByte);
+    return static_cast<int16_t>((firstByte << 8) | secondByte);
```

Comment:
:warning: **readability-magic-numbers** :warning:
8 is a magic number; consider replacing it with a named constant